lib/shared.js

```diff
@@ -191,5 +191,13 @@ module.exports = {
 				'@typescript-eslint/no-empty-function': ['error', { allow: ['arrowFunctions'] }],
 			},
 		},
+
+		{
+			files: ['**/constants.ts'],
+			rules: {
+				// allow named exports in constants files
+				'import/prefer-default-export': 'off',
```

Comment:
The `node` config is doing something similar for `constants.ts` and with a tweak here it won't have to any longer:

```js
		{
			files: [
				'**/*.types.ts',
				'**/types/*.ts',
				'**/constants.ts',
				'**/*.schema.ts',
				'**/instances/**',
			],
			rules: {
				'import/prefer-default-export': 'off',
				'import/no-default-export': 'error',
			},
		},
```

If you were to add: `'import/no-default-export': 'error',` to this override, `node` wouldn't have to include `**/constants.ts` in  its override because it would be covered by this `shared` config.

Reply:
I concur here.  We should definitely bias towards having the same config wherever possible so the more we can migrate into shared and not have a representation in the individual configs the better.

Reply:
I pushed up a commit with these changes. Thanks!